django_geosource/tests/test_api.py

```diff
@@ -40,6 +72,36 @@ def test_list_view(self):
         self.assertEqual(response.status_code, HTTP_200_OK)
         self.assertEqual(Source.objects.count(), len(response.json()))
 
+    def test_refresh_view_fail(self):
+        source = GeoJSONSource.objects.create(
+            name="test",
+            geom_type=GeometryTypes.Point.value,
+            file=os.path.join(os.path.dirname(__file__), "data", "test.geojson"),
+        )
```

Comment:
This source is used in a lot of test methods, you should put it in `setUp()` as an attribute:

```
this.source = GeoJSONSource.objects.create(
    name="test",
    geom_type=GeometryTypes.Point.value,
    file=os.path.join(os.path.dirname(__file__), "data", "test.geojson"),
)

Reply:
Done